docs/installation.rst

```diff
@@ -25,6 +25,9 @@ Installation on your system
   * ``cd pyradiomics``
   * ``sudo python -m pip install -r requirements.txt``
   * ``sudo python setup.py install``
+  
+  To use your build for interactive use and development:
+  * ``sudo python setup.py develop``
```

Comment:
@pieper, I'll merge this, and then remove the `sudo`'s in the update documentation PR (#233).